docs/ERC-725.md

```diff
@@ -196,6 +225,8 @@ interface IERC725Y /* is ERC165, ERC173 */ {
 
     function getData(bytes32[] memory _keys) external view returns(bytes[] memory);
```

Comment:
We might want to stick to a convention for the parameter names.
Either prefixed with underscore `_` or not.

In the interface contract `IERC725Y.sol`, they are not with underscore.

In `ERC725YCore.sol` too except for `setData(bytes32[] _keys, bytes[] _values)`.
https://github.com/YamenMerhi/ERC725/blob/e0665aafa9c98109037ed799e6a58e96a8d95d35/implementations/contracts/ERC725YCore.sol#L67